torch/testing/_internal/common_methods_invocations.py

```diff
@@ -178,9 +178,9 @@ class AliasInfo(object):
     torch.absolute, torch.Tensor.absolute, torch.Tensor.absolute_
     """
 
-    def __init__(self, alias_name):
+    def __init__(self, alias_name, alias_op=None):
         self.name = alias_name
-        self.op = _getattr_qual(torch, alias_name)
+        self.op = alias_op if alias_op else _getattr_qual(torch, alias_name)
```

Comment:
Nice extension

Reply:
Apologies, earlier we decided to keep this to avoid extra OpInfo entry for `special` alias. Later, we decided to instead have an extra OpInfo entry since it's only required for ops like polygamma which need re-ordering of the arguments.

This has been removed now.